src/components/link/link.html

```diff
@@ -1 +1,2 @@
 <a href="#" class="bx--link">Link</a>
+<a href="#" class="bx--link" tabindex="-1" aria-disabled="true" disabled>Link</a>
```

Comment:
`disabled` is not a valid attribute for an `<a>` element,  only `aria-disabled` should be used.

Reply:
And add a `bx--link--disabled` class instead?

Reply:
Ah right, was just using it to target it via css, but figured I can use `&[aria-disabled="true"]` instead